final_task/rss_reader/work_with_html.py

```diff
@@ -0,0 +1,30 @@
+import os
+import datetime
+
+
+def write_to_html_file(data: dict, path: str):
+    if os.path.isdir(path):
+        date_str = datetime.datetime.now().date().strftime('%Y%m%d')
+        filename = path + os.path.sep + date_str + '_' + data['title'][:data['title'].find(' ')].replace(':', '') + '.html'
```

Comment:
it is better to just use `os.path.join` instead of manually concatenate multiple strings into one using `os.path.sep`